pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -165,46 +166,41 @@ func (r *Reconciler) reconcile(ctx context.Context, source *v1alpha1.ApiServerSo
 		r.Logger.Error("Unable to create the receive adapter", zap.Error(err))
 		return err
 	}
-	// Update source status
 	source.Status.PropagateDeploymentAvailability(ra)
+
+	track := r.resourceTracker.TrackInNamespace(source)
 	if err = track(utils.ObjectRef(ra, deploymentGVK)); err != nil {
```

Comment:
Based on #1998, I recommend not using this tracker. Instead have controller.go setup the informer, like https://github.com/knative/eventing/blob/master/pkg/reconciler/broker/controller.go#L94-L97

Reply:
ah good point, thx!